lib/test/structuredfield/util/id3Generator.ts

```diff
@@ -0,0 +1,114 @@
+import { toUint8 } from '../../../src/id3/util/utf8.js';
+
+export function generateId3(
+	frames: Uint8Array,
+	extendedHeader: boolean = false
+) {
+	let result = concat(
+		stringToInts('ID3'),
```

Comment:
This file is in the `structuredfield` folder. Please move it to `lib/test/id3/util/`